### PR TITLE
Add public_url table

### PR DESCRIPTION
**Pull request by Coteh**
Add a schema for public URLs to the database. This is for #299 and is an incremental change to address #235.

### CODE REVIEW

File: backend/app/adapter/migration/8_create_public_url_table.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-- +migrate Up
+CREATE TABLE public_url (
+    alias CHARACTER VARYING,
```
Comment (byliuyang) [inline backend/app/adapter/migration/8_create_public_url_table.sql]:
```suggestion
    alias CHARACTER VARYING(50)  NOT NULL,
```

**Comment by Coteh** [comment]
@byliuyang That makes sense to have it as `varchar(50)`, as `user_url_relation` also has the same restriction on `alias` column. Thanks for the suggestion! Should the `alias` column in `url` table be updated to have this restriction as well?

**Comment by byliuyang** [comment]
Hi @Coteh , yes. Alias in url table should have the same restriction. Could you please open an issue to propose this change? Thanks!

**Comment by Coteh** [comment]
@byliuyang Issue created. #316. Also going to close #299.

**Pull request merged**